packages/common-db/src/lib/common/tools.ts

```diff
@@ -18,16 +18,20 @@ export function statesDbHasServer(dbType: string): boolean {
 /**
  * Allows to find out if a given objects dbType offers a server which runs on this host and listens (locally or globally/by IP)
  * @param dbType database type
- * @param host configured db host
+ * @param host configured db host - multihost (array) will always return false
```

Comment:
You could model that with an overload if desirable.

Reply:
Not sure if necessary, only runtime really knows what is passed in this case, so we are only interested that it is a boolean return type.